modules/shadertools/src/modules/lighting/pbr-material/pbr-material.ts

```diff
@@ -13,52 +13,21 @@ import {lighting} from '../lights/lighting-uniforms';
 
 import {vs} from './pbr-vertex-glsl';
 import {fs} from './pbr-fragment-glsl';
-
-export type PBRMaterialProps = PBRMaterialBindings & {
-  unlit: boolean;
-
-  // Base color map
-  baseColorMapEnabled: boolean;
-  baseColorFactor: Readonly<Vector4 | NumberArray4>;
-
-  normalMapEnabled: boolean;
-  normalScale: number; // #ifdef HAS_NORMALMAP
-
-  emissiveMapEnabled: boolean;
-  emissiveFactor: Readonly<Vector3 | NumberArray3>; // #ifdef HAS_EMISSIVEMAP
-
-  metallicRoughnessValues: Readonly<Vector2 | NumberArray2>;
-  metallicRoughnessMapEnabled: boolean;
-
-  occlusionMapEnabled: boolean;
-  occlusionStrength: number; // #ifdef HAS_OCCLUSIONMAP
-
-  alphaCutoffEnabled: boolean;
-  alphaCutoff: number; // #ifdef ALPHA_CUTOFF
-
-  // IBL
-  IBLenabled: boolean;
-  scaleIBLAmbient: Readonly<Vector2 | NumberArray2>; // #ifdef USE_IBL
-
-  // debugging flags used for shader output of intermediate PBR variables
-  // #ifdef PBR_DEBUG
-  scaleDiffBaseMR: Readonly<Vector4 | NumberArray4>;
-  scaleFGDSpec: Readonly<Vector4 | NumberArray4>;
-};
+import {pbrProjection} from './pbr-projection';
 
 /** Non-uniform block bindings for pbr module */
-type PBRMaterialBindings = {
+export type PBRMaterialBindings = {
   // Samplers
-  baseColorSampler?: Texture | null; // #ifdef HAS_BASECOLORMAP
-  normalSampler?: Texture | null; // #ifdef HAS_NORMALMAP
-  emissiveSampler?: Texture | null; // #ifdef HAS_EMISSIVEMAP
-  metallicRoughnessSampler?: Texture | null; // #ifdef HAS_METALROUGHNESSMAP
-  occlusionSampler?: Texture | null; // #ifdef HAS_OCCLUSIONMAP
+  u_BaseColorSampler?: Texture | null; // #ifdef HAS_BASECOLORMAP
```

Comment:
Are we going with `u_` for bindings (even though bindings are not uniforms?). I have been gravitating toward module name prefixes `pbr_baseColorSampler`...

Reply:
I've [done a pass](https://github.com/visgl/luma.gl/pull/2173/commits/94ef7e1586b2a8a9a6c6a17a6772dc05b4027f57) to rename everything according to this convention